File: WebApp/Front/src/models/Subject.ts
import Teacher from "./Teacher";

export default class Subject {
  subjectId: number;
  subjectName: string;
  hourNumber: number;
  teacherDto: Teacher;

  constructor(
    id: number,
    subjectName: string,
    hourNumber: number,
    teacher: Teacher
  ) {
    this.subjectId = id;
    this.subjectName = subjectName;
    this.hourNumber = hourNumber;
    this.teacherDto = teacher;
  }
}

File: face_recognition/models/Subject.py

from models.Teacher import Teacher

class Subject: 
  def __init__(self, subjectId: int, subjectName: str, hourNumber: int, teacherDto: Teacher):
    self.subjectId = subjectId
    self.subjectName = subjectName
    self.hourNumber = hourNumber
    self.teacherDto = teacherDto
